IapExample/yarn.lock

```diff
@@ -7002,7 +7002,7 @@ __metadata:
     jest: ^29.0.0
     lodash: 4.17.21
     metro-react-native-babel-preset: ^0.76.0
-    react: 17.0.2
+    react: 18.2.0
```

Comment:
Nice upgrade but it looks like this should be `18.0.0` based on https://react-native-community.github.io/upgrade-helper/?from=0.68.2&to=0.69.2